### PR TITLE
Fix Some CompilerServices implemenatations

### PR DESCRIPTION
**Pull request by ramarag**
@tarekms @weshaggard  @stephentoub 

There were couple of issues regarding the compilerservices types not getting type forwarded correctly. This change should fix it.

### CODE REVIEW

File: src/System.Runtime/src/System/Runtime/CompilerServices/StrongBox.cs [hunk 1]
```diff
@@ -56,4 +56,10 @@ public interface IStrongBox
         /// </summary>
         object Value { get; set; }
     }
+    public static partial class IsConst
+    {
+    }
+    public static partial class IsVolatile
+    {
+    }
```
Comment (stephentoub) [inline src/System.Runtime/src/System/Runtime/CompilerServices/StrongBox.cs]: Why are these being added to StrongBox.cs?

File: src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComAwareEventInfo.cs [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+        public IUnknownConstantAttribute() { }
+        public override object Value
+        { 
+            get 
+            { 
+                return new UnknownWrapper(null); 
+            } 
+        }
+    }
+}
```
Comment (stephentoub) [inline src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComAwareEventInfo.cs]: Why is this type being added to ComAwareEventInfo.cs?  Shouldn't it be added to its own file?  It's not even in the same namespace :)

File: src/System.Runtime.CompilerServices.VisualC/src/System/Runtime/CompilerServices/Attributes.cs [hunk 3]
```diff
@@ -57,9 +54,6 @@ public static partial class IsUdtReturn
     {
     }
     [System.Runtime.InteropServices.ComVisibleAttribute(true)]
-    public static partial class IsVolatile
```
Comment (stephentoub) [inline src/System.Runtime.CompilerServices.VisualC/src/System/Runtime/CompilerServices/Attributes.cs]: The type is being removed, but the attribute that was applied to it wasn't deleted?

File: src/System.Runtime/src/System/Runtime/CompilerServices/StrongBox.cs (hunk 1 above)
Comment (weshaggard) [inline src/System.Runtime/src/System/Runtime/CompilerServices/StrongBox.cs]: I agree they shouldn't be added here.

Comment (ramarag) [inline src/System.Runtime/src/System/Runtime/CompilerServices/StrongBox.cs]: done

File: src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComAwareEventInfo.cs (hunk 2 above)
Comment (ramarag) [inline src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComAwareEventInfo.cs]: moved it to a new file

File: src/System.Runtime.CompilerServices.VisualC/src/System/Runtime/CompilerServices/Attributes.cs (hunk 3 above)
Comment (ramarag) [inline src/System.Runtime.CompilerServices.VisualC/src/System/Runtime/CompilerServices/Attributes.cs]: removed

**Comment by tarekgh** [comment]
LGTM

**Comment by ramarag** [comment]
@dotnet-bot test  Innerloop Windows_NT Release Build and Test

**Pull request merged**